R/classify.R

```diff
@@ -0,0 +1,101 @@
+#' Possible subtype labels for MB samples
+subtype_labels <- c("WNT", "SHH", "G3", "G4")
+
+
+#' Classify a set of samples using the unweighted KSTP classifier
+#'
+#' This function wraps the `multiclassPairs::predict_one_vs_rest_TSP()` function
+#' to run the unweighted KSTP classifier on a provided matrix of samples.
```

Comment:
```suggestion
#' to run the unweighted KSTP classifier on a provided gene expression matrix of samples.
```
Just adding a clarifying comment that the "matrix of samples" should contain gene expression data.